skimage/draw/_polygon2mask.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+from .. import draw
+
+
+def polygon2mask(image_shape, polygon):
+    """Compute a mask from polygon.
+
+    Parameters
+    ----------
+    image_shape : tuple of size 2.
+        The shape of the mask.
+    polygon : array_like.
+        The polygon coordinates of shape (N, 2) where N is
+        the number of points.
+
+    Returns
+    -------
+    mask : 2-D ndarray of type 'bool'.
+        The mask that corresponds to the input polygon.
+
+    Notes
+    -----
+    This function does not do any border checking, so that all
+    the vertices need to be within the given shape.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skimage.transform import polygon2mask
```

Comment:
you forgot that one @hadim  :)

Reply:
Wait. Are the examples in docstrings executed? Wow!